### ISSUE TITLE
No shortcut key for superscript and subscript

### ISSUE THREAD

**Issue by rmaheshkumarblr** [HappenApps/Quiver#621]
When working with mathematical expressions, it would be easier if there was shortcut key for subscript!

**Comment by MyGodItsFull0fStars** [comment]
I agree! Often I don't need the full expression power of LaTeX and just want to squeeze a few Sub/Superscripts in a Text Cell. Having shortcuts for both would be a great addition.

**Comment by jooohhn** [comment]
Hopefully this still gets implemented!
